pkg/promtail/targets/filetargetmanager.go

```diff
@@ -199,72 +207,72 @@ func (s *targetSyncer) sync(groups []*targetgroup.Group) {
 	dropped := []Target{}
 
 	for _, group := range groups {
-		for _, t := range group.Targets {
-			level.Debug(s.log).Log("msg", "new target", "labels", t)
-
-			discoveredLabels := group.Labels.Merge(t)
-			var labelMap = make(map[string]string)
-			for k, v := range discoveredLabels.Clone() {
-				labelMap[string(k)] = string(v)
-			}
+		host, ok := group.Labels[hostLabel]
+		if ok && string(host) != s.hostname {
```

Comment:
Should we get rid of this ? It seems that now we only get target for the given host ? This was only used for Kubernetes SD